### DISCUSSION TITLE
Welcome to sfdc-lwc-lightning-datatable Discussions!

### DISCUSSION THREAD

**Discussion by Sarveshgithub** [Sarveshgithub/sfdc-lwc-lightning-datatable#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trusha82** [comment]
Hi Sarvesh,
Thank you for building this component. 
I am wondering whether it will work on the rich text box column. I have currently added to my lightning page and the content is not displaying properly. Please see the attached image. 
I have noticed one more issue (maybe I haven't set it up properly). while entering search text, it displays other records (e.g. fetching other case file's notes) data too. I have placed this component on Case Files where It displays all the related case notes based on case file number. 

Let me know what should I have to do.
Thanks!
Trusha

![Case Notes Screen](https://github.com/Sarveshgithub/sfdc-lwc-lightning-datatable/assets/52017492/df43673e-b72a-4200-a43d-75897fa4609f)
